pkg/agent/openflow/pipeline.go

```diff
@@ -279,6 +308,18 @@ func (c *client) l2ForwardOutputFlow(category cookie.Category) binding.Flow {
 		Done()
 }
 
+// l2ForwardOutputFlow generates the flow that forward peer node traffic via gw0.
+// This flow supersedes default output flow because ovs by default auto-skips packets with output = input port.
+func (c *client) l2ForwardOutputToRemoteInPortOut(gwPort uint32, category cookie.Category) binding.Flow {
```

Comment:
The name in comment is still `l2ForwardOutputFlow`.
The function name is a little hard to understand, how about `l2ForwardOutputInPortFlow` given it's a special case of `l2ForwardOutputFlow`.

Reply:
done